views/js/qtiCreator/widgets/interactions/extendedTextInteraction/states/Question.js

```diff
@@ -71,10 +71,11 @@ define([
         }
 
 
-        var patternMask = interaction.attr('patternMask');
-        var expectedLength = parseInt(interaction.attr('expectedLength'), 10);
-        var maxWords = parsePattern(patternMask,'words');
-        var maxChars = parsePattern(patternMask,'chars');
+        var patternMask = interaction.attr('patternMask'),
+        expectedLength = parseInt(interaction.attr('expectedLength'), 10),
+        expectedLines = parseInt(interaction.attr('expectedLines'),10),
+        maxWords = parsePattern(patternMask,'words'),
+        maxChars = parsePattern(patternMask,'chars');
```

Comment:
parsePattern() causes js error if the typed patternMask is incorrect ... I know this is a tricky issue to validate the pattern mask provided by the user, but I believe we should provide at least some fault tolerance or allow it to fail gracefully. What do you think about that ?